File: c++/control_flow/if_statement/if_statement_formatting.cpp
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */
// SPDX-License-Identifier: GPL-3.0-or-later

#include <iostream>

int main()
{
  bool condition;

  condition = (1 > 2);

  if (condition)
    std::cout << "1 > 2" << std::endl;

  condition = (2 > 1);

  if (condition)
    std::cout << "2 > 1" << std::endl;

  std::cout << std::endl;

  if (1 > 2)
    std::cout << "1 > 2" << std::endl;

  if (2 > 1)
    std::cout << "2 > 1" << std::endl;

  std::cout << std::endl;

  if (1 > 2) std::cout << "1 > 2" << std::endl;

  if (2 > 1) std::cout << "2 > 1" << std::endl;

  std::cout << std::endl;

  if (1 > 2)
  {
    std::cout << "1 > 2" << std::endl;
  }

  if (2 > 1)
  {
    std::cout << "2 > 1" << std::endl;
  }

  std::cout << std::endl;
}

File: python/control_flow/if_statement/if_statement_formatting.py
# Copyright (C) 2019-2026 Katarzyna Krasińska
# Languagium - https://github.com/katheroine/languagium
# Licensed under GPL-3.0 - see LICENSE.md
# SPDX-License-Identifier: GPL-3.0-or-later

condition = 1 > 2

if condition:
    print('1 > 2')

condition = 2 > 1

if condition:
    print('2 > 1')

print()

if 1 > 2:
    print('1 > 2')

if 2 > 1:
    print('2 > 1')

print()

if 2 < 1: print('2 > 1')

if 2 > 1: print('2 > 1')

print()
